### ISSUE TITLE
Datetime fromisoformat method is not supported in python 3.6 

### ISSUE THREAD

**Issue by Zerlok** [maximdanilchenko/aiochclient#11]
A few days earlier I've caught the bug with my services on python 3.6, which uses async clickhouse client:

```python
Traceback (most recent call last):
  File "aiochclient/_types.pyx", line 13, in aiochclient._types
ModuleNotFoundError: No module named 'ciso8601'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  ...
  File "/requirements/aiochclient/__init__.py", line 1, in <module>
    from aiochclient.client import ChClient
  File "/requirements/aiochclient/client.py", line 8, in <module>
    from aiochclient.records import Record, RecordsFabric
  File "/requirements/aiochclient/records.py", line 6, in <module>
    from aiochclient._types import what_py_converter
  File "aiochclient/_types.pyx", line 19, in init aiochclient._types
AttributeError: type object 'datetime.datetime' has no attribute 'fromisoformat'
```

According to the [doc page](https://docs.python.org/3/library/datetime.html#datetime.date.fromisoformat), the method was introduced in python 3.7, so now this code breaks compatibility with 3.6 versions.

**Comment by maximdanilchenko** [comment]
Hi! Thank you for the issue. I will fix it today.

For now you can use [speedups] option to install it with ciso8601 lib (or just install ciso8601 to the same environment) - aiochclient will use it instead of std datetime. Or use older versions.

**Comment by Zerlok** [comment]
Hi! Thanks for quick response! Ok, I'll try your suggestion

**Comment by maximdanilchenko** [comment]
Fixed in 1.1.4
